tests/e2e/initialization/config.go

```diff
@@ -347,10 +355,35 @@ func updateMintGenesis(mintGenState *minttypes.GenesisState) {
 
 func updateTxfeesGenesis(txfeesGenState *txfeestypes.GenesisState) {
 	txfeesGenState.Basedenom = OsmoDenom
+	txfeesGenState.Feetokens = []txfeestypes.FeeToken{
+		{Denom: E2EFeeToken, PoolID: 1},
+	}
 }
 
 func updateGammGenesis(gammGenState *gammtypes.GenesisState) {
 	gammGenState.Params.PoolCreationFee = tenOsmo
+	// setup fee pool, between "e2e_default_fee_token" and "uosmo"
+	feePoolParams := balancer.NewPoolParams(sdk.MustNewDecFromStr("0.01"), sdk.ZeroDec(), nil)
+	feePoolAssets := []balancer.PoolAsset{
+		{
+			Weight: sdk.NewInt(100),
+			Token:  sdk.NewCoin("uosmo", sdk.NewInt(100000000000)),
+		},
+		{
+			Weight: sdk.NewInt(100),
+			Token:  sdk.NewCoin(E2EFeeToken, sdk.NewInt(100000000000)),
+		},
+	}
+	pool1, err := balancer.NewBalancerPool(1, feePoolParams, feePoolAssets, "", time.Unix(0, 0))
+	if err != nil {
+		panic(err)
+	}
+	anyPool, err := types1.NewAnyWithValue(&pool1)
+	if err != nil {
+		panic(err)
+	}
+	gammGenState.Pools = []*types1.Any{anyPool}
+	gammGenState.NextPoolNumber = 2
```

Comment:
Can we remove this and instead re-create this via CLI?

This interferes with pool IDs that we are creating via CLI

Reply:
Two questions:
- Why does it interfere?
- Where do it in CLI such that it comes first? It needs to be first so that other txs can pay fees, and we'd have to make distinct custom methods for paying fees to set this up.

Reply:
1. The reason is that we also create pre-upgrade pools as a result the pool is of 1 is already assigned after upgrade. It is possible to work around this by first making a separate PR to v14 genesis.

2. Makes sense. I will work around number 1 then and make the changes to v14 before main

Reply:
Actually, for 2, it should be possible to create a pool with the fee via pool file, right?

Reply:
The pool file doesn't contain the tx fee. (Adding something custom for the tx fee for initialization isn't really a big deal. It just has to come first, if you point me to where I do that, I can add it in relatively easily)